src/mito2/src/engine.rs

```diff
@@ -95,6 +95,11 @@ impl MitoEngine {
     pub(crate) fn get_region(&self, id: RegionId) -> Option<crate::region::MitoRegionRef> {
         self.inner.workers.get_region(id)
     }
+
+    /// Returns the estimated size of the region's WAL.
+    pub fn get_region_wal_size(&self, id: RegionId) -> Result<usize> {
```

Comment:
non-blocking: Well, I think we can return a struct that contains the WAL size, memtable size, manifest size and SST size of the region in the future. Then we don't need this method.

Reply:
I will remove this method